### PR TITLE
E2E test for EM-DAT

### PR DESCRIPTION
**Pull request by Shhhhhubh**
Addresses
- Added test case for EM-DAT along with necessary edge cases test files.

## Changes

* Added Test case file 
* Added Edge case json5 file 
* Added Fixed output file for comparision 

## This PR doesn't introduce any:

- [ ] temporary files, auto-generated files or secret keys
- [ ] n+1 queries
- [ ] linter issues
- [ ] `print`
- [ ] typos
- [ ] unwanted comments

## This PR contains valid:

- [ ] tests

### CODE REVIEW

File: gh-docker-compose.yml [hunk 1]
```diff
@@ -7,6 +7,8 @@ services:
       CI: "true"
     volumes:
       - ./coverage/:/code/coverage/
-
+  geocoding-service:
+    image: ghcr.io/ifrcgo/geocoding-service:main.99b10c8
+    command: /bin/sh -c "uvicorn service:app --host 0.0.0.0 --port 8001 --reload --workers 1"
```
Comment (ranjan-stha) [inline gh-docker-compose.yml:12]: @Shhhhhubh instead of hard coded `port`, can we get it from the env variable?

File: apps/etl/tests/sources/emdat_test.py [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+        "output": "output_emdat.json",
+        "expected": "fixed_output_emdat.json",
+    },
+]
+
+
+@pytest.mark.django_db
+@pytest.mark.parametrize("case", TEST_CASES)
+@override_settings(
+    TRANSFORM_SUCCESS_RATE=20,
```
Comment (ranjan-stha) [inline apps/etl/tests/sources/emdat_test.py:33]: @Shhhhhubh do we still have to use 20% as success rate? The success rate is available from env var. Please check.

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -23,7 +23,7 @@ env:
   COMPOSE_FILE: docker-compose.yml:gh-docker-compose.yml
   DJANGO_SECRET_KEY: "ci-test-insecure-django-secret-key"
   # ETL app configs
-  GEOCODER_URL: https://geocoder.dummy.com
+  GEOCODER_URL: http://geocoding-service:8001
```
Comment (ranjan-stha) [inline .github/workflows/ci.yml:26]: @Shhhhhubh Do we hard code the port?

**Review by sudan45 — APPROVED** [review]
